### ISSUE TITLE
Way to make icons smaller than the smallest?

### ISSUE THREAD

**Issue by cattenbb** [MrC0rrupted/WinLaunch#33]
I would love to be able to make the icons smaller. They look giant on my screen even with the icon size turned down to the smallest size.

**Comment by GreenskyDan** [comment]
Yes agreed. I am using this because windows 11 removed the fullscreen tile support. I used to have 84 tiles on my screen. Even with the smallest option for icon/grid size, it's still too big. It would be nice if there was an option to use smaller tiles (without app names) because usually the icon is enough for me.
